mysql/tests/conftest.py

```diff
@@ -4,44 +4,16 @@
 import subprocess
 import time
 import os
+import sys
 
 import pytest
 import pymysql
-import logging
 
-import common
+from . import common
 
-log = logging.getLogger('test_mysql')
 
-
-def wait_for_mysql():
-    """
-    Wait for the slave to connect to the master
-    """
-    connected = False
-    for i in xrange(0, 100):
-        try:
-            pymysql.connect(
-                host=common.HOST,
-                port=common.PORT,
-                user=common.USER,
-                passwd=common.PASS,
-                connect_timeout=2
-            )
-            pymysql.connect(
-                host=common.HOST,
-                port=common.SLAVE_PORT,
-                user=common.USER,
-                passwd=common.PASS,
-                connect_timeout=2
-            )
-            connected = True
-            return True
-        except Exception as e:
-            log.debug("exception: {}".format(e))
-            time.sleep(2)
-
-    return connected
+MYSQL_FLAVOR = os.getenv('MYSQL_FLAVOR', 'mysql')
```

Comment:
Why not have the version here too?

Reply:
It's used only once so I thought it didn't deserve to be a module-global